openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -346,18 +337,12 @@ def user_timezone(self):
 
     @property
     def can_access_proctored_exams(self):
-        """Returns if the user is eligible to access proctored exams"""
-        if is_masquerading_as_non_audit_enrollment(
```

Comment:
Now staff could masquerade as audit user, why do that functionality is removed?

Reply:
The logic stays the same here. Masquerade for a particular mode is not available anymore (that was a part of enrollment_track partition), so we just check the enrollment type